tests/test_output.py

```diff
@@ -0,0 +1,138 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytest
+import xarray as xr
+
+from climate_index_collection.data_loading import VARNAME_MAPPING, load_data_set
+from climate_index_collection.indices import (
+    north_atlantic_oscillation,
+    southern_annular_mode,
+)
+from climate_index_collection.output import (
+    compute_index,
+    concat_indices,
+    index_dataarray_to_dataframe,
+    run_full_workflow_csv,
+)
+
+
+# Dummy example data
+#
+# data = [0.5, 1, -0.3, -0.7]
+# dim: only 'time'
+# name: 'SAM'
+
+
+@pytest.fixture
+def example_data_array():
+    data = xr.DataArray(
+        [0.5, 1, -0.3, -0.7],
+        dims=("time"),
+        name="SAM",
+    )
+    return data
```

Comment:
BTW, I really like that you follow best practices here and set up a minimal data set!